# HG changeset patch
# Parent 1e8fc570ee358eef9986a6756c5f61402b48c2fb
# User Ben Kelly <ben@wanderview.com>
Alter the Android package name to "Fennec SW" to avoid overwriting other installs.

diff --git a/mobile/android/branding/nightly/configure.sh b/mobile/android/branding/nightly/configure.sh
--- a/mobile/android/branding/nightly/configure.sh
+++ b/mobile/android/branding/nightly/configure.sh
@@ -1,7 +1,8 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-MOZ_APP_DISPLAYNAME=Nightly
-MOZ_UPDATER=1
-MOZ_ANDROID_ANR_REPORTER=1
+ANDROID_PACKAGE_NAME=org.mozilla.fennec_sw
+MOZ_APP_DISPLAYNAME="Fennec SW"
+MOZ_UPDATER=
+MOZ_ANDROID_ANR_REPORTER=
diff --git a/mobile/android/branding/unofficial/configure.sh b/mobile/android/branding/unofficial/configure.sh
--- a/mobile/android/branding/unofficial/configure.sh
+++ b/mobile/android/branding/unofficial/configure.sh
@@ -1,8 +1,8 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-ANDROID_PACKAGE_NAME=org.mozilla.fennec_`echo $USER | sed 's/-/_/g'`
-MOZ_APP_DISPLAYNAME="Fennec `echo $USER | sed 's/-/_/g'`"
+ANDROID_PACKAGE_NAME=org.mozilla.fennec_sw
+MOZ_APP_DISPLAYNAME="Fennec SW"
 MOZ_UPDATER=
 MOZ_ANDROID_ANR_REPORTER=
